# result timestamp:
time stamp
# device name:
string device_name
# number of IMU observations collected:
uint32 num_observations
# number of clusters used for Allan variance params estimation:
uint32 num_clusters

# estimated parameters:
imu_calibration/AllanVarianceParams[] noises